src/find_libpython/__init__.py

```diff
@@ -131,6 +131,23 @@ def wrapper(*args, **kwds):
     return wrapper
 
 
+def _get_proc_library(basename=False):
+    pid = os.getpid()
+    path = f"/proc/{pid}/maps"
+    _logger.debug(f"Checking {path} for libpython")
+    lines = open(path).readlines()
+
+    for l in lines:
+        path = l.split(" ")[-1].lstrip().rstrip()
```

Comment:
This doesn't work if the path has a space in it.
```suggestion
        path = l.split(" ", 5)[5].strip()
```